### PR TITLE
Update migration documentation for node.js adapter about logout

### LINKED ISSUE #1453: Update migration documentation for node.js adapter about logout
### Description

Regarding to changes in the RP-Initiated Logout in Keycloak main and related changes in the node.js adapter (more details in the https://github.com/keycloak/keycloak-nodejs-connect/pull/359 ), there is a need to adjust documentation a bit.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
Closes #1453

### CODE REVIEW

**Comment by mposolda** [comment]
@jonkoops @andymunro Do you please have a chance to review this?

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 1]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
   and that one was not changed.
 - If you use the latest javascript adapter, you are also not affected. However if your application uses an older version of the JavaScript adapter, you are affected as this
   adapter uses the variant of the logout endpoint with the deprecated `redirect_uri` parameter. In this case, you may need to upgrade to the latest version of the JavaScript adapter.
-  The same applies for the Node.js adapter.
+- For the node.js adapter, same applies as for the javascript adapter. You are encouraged to update to the latest version as the older version of the adapter uses the deprecated `redirect_uri` parameter.
```
Comment (jonkoops) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]:
```suggestion
- For the Node.js adapter, same applies as for the JavaScript adapter. You are encouraged to update to the latest version as the older version of the adapter uses the deprecated `redirect_uri` parameter.
```

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 2]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
...
+  With the latest node.js adapter, you are not affected as long as you use the logout based on the `/logout` URL as described in the documentation or in the node.js adapter example. However in the case
```
Comment (jonkoops) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]:
```suggestion
  With the latest Node.js adapter, you are not affected as long as you use the logout based on the `/logout` URL as described in the documentation or in the Node.js adapter example. However in the case
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by mposolda** [comment]
@jonkoops Applied your suggestions. Thanks!

**Review by jonkoops — DISMISSED** [review]
LGTM

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 3]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
   and that one was not changed.
 - If you use the latest javascript adapter, you are also not affected. However if your application uses an older version of the JavaScript adapter, you are affected as this
   adapter uses the variant of the logout endpoint with the deprecated `redirect_uri` parameter. In this case, you may need to upgrade to the latest version of the JavaScript adapter.
-  The same applies for the Node.js adapter.
+- For the Node.js adapter, same applies as for the JavaScript adapter. You are encouraged to update to the latest version as the older version of the adapter uses the deprecated `redirect_uri` parameter.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]: the same guideline

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 4]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
...
+  With the latest Node.js adapter, you are not affected as long as you use the logout based on the `/logout` URL as described in the documentation or in the Node.js adapter example. However in the case
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]: Add a comma after However,

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 5]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
...
+  when your application directly uses method `keycloak.logoutUrl`, you can consider adding `idTokenHint` as the second argument to this method. The possibility to add `idTokenHint` as second argument was newly
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]: add "the" before "method"

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 6]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
...
+  added in this version. The `idTokenHint` needs to be valid ID Token, which was obtained during the login. Adding `idTokenHint` is optional and  If you don't add it, your users will need to confirm logout
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]: needs to be a valid ID Token that was obtained during the login

Use that (without a preceding comma) when the following phrase qualifies the earlier phrase --- it is the specifically the token that was obtained during the login.
Use ", which" to begin a phrase that does not qualify the earlier phrase.

File: upgrading/topics/keycloak/changes-18_0_0.adoc [hunk 7]
```diff
@@ -35,7 +35,11 @@ The existing deployments are affected in the following ways:
...
+  screen as described above. Also they won't be redirected back to the application after logout.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-18_0_0.adoc]: change " optional and  If you don't add it, your users will need to confirm logout   screen as described above. Also they won't be redirected back to the application after logout."
TO
"optional, but if you omit it, your users will need to confirm the logout screen as described earlier. Also they will not be redirected back to the application after logout."

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @mposolda. A few suggested changes.

File: upgrading/topics/keycloak/changes-18_0_0.adoc (hunk 6 above)
Comment (mposolda) [reply upgrading/topics/keycloak/changes-18_0_0.adoc]: Thanks for the hint!

**Comment by mposolda** [comment]
@andymunro I've applied all your suggestions. Thanks for the review!

**Review by jonkoops — APPROVED** [review]

**Pull request merged**